### ISSUE TITLE
Remove dependency on a custom engine fork

### ISSUE THREAD

**Issue by stechyo** [stechyo/godot-steam-audio#18]
Obviously I'd like to not make every user rely on downloading some guy's fork of Godot to use this extension. However, this is not feasible right now. This issue is blocked by:

https://github.com/godotengine/godot-proposals/issues/8609
https://github.com/godotengine/godot/pull/86539 (or another pull request that does this in a better way)

**Comment by fire** [comment]
Is this still valid?

**Comment by stechyo** [comment]
Yeah.

**Comment by octanejohn** [comment]
this can close i think, is going to be in godot 4.4

**Comment by fire** [comment]
As far as I can tell all the godot-steam-audio documentation says it requires a godot fork.

https://github.com/stechyo/godot/commit/231d764ea1eb895359c7ce7cac391d5613618f7e

**Comment by octanejohn** [comment]
this pr was merged https://github.com/godotengine/godot/pull/86539

**Comment by fire** [comment]
Cheers

**Comment by stechyo** [comment]
Yeah! Closing this.

**Comment by TheIndiv** [comment]
Wait, so a custom fork is no longer required?

**Comment by stechyo** [comment]
It is not required provided you get the release on the PR, and you are using Godot 4.4. When there is a stable release for 4.4 I’ll merge the PR and release a new version of the extension.

**Comment by DevPrice** [comment]
@stechyo any update here now that 4.4 has dropped?

**Comment by stechyo** [comment]
Well, the update is that the issue is closed, we no longer need it

**Comment by DevPrice** [comment]
Cool! Would be useful to update the installation docs then, since they still say the fork is required:
https://github.com/stechyo/godot-steam-audio/wiki/Installation

**Comment by stechyo** [comment]
Done, thanks for the callout!
